module.txt

```diff
@@ -6,8 +6,11 @@
     "description" : "A collection of various structure templates.",
     "dependencies" : [
     	{"id": "Core", "minVersion": "2.1.1-SNAPSHOT"},
+    	{"id": "CoreAssets", "minVersion": "1.0.1"},
+    	{"id": "CoreBlocks", "minVersion": "1.0.1"},
     	{"id": "StructureTemplates", "minVersion": "0.3.0-SNAPSHOT"},
-    	{"id": "Fences", "minVersion": "0.1.1-SNAPSHOT"}
+    	{"id": "Fences", "minVersion": "0.1.1-SNAPSHOT"},
+    	{"id": "StructuralResources", "minVersion": "0.1.1-SNAPSHOT"}
```

Comment:
This won't actually work: needs to be `2.0.0` which is the latest version of StructuralResources